charts/wildfly/values.yaml

```diff
@@ -110,7 +110,7 @@ wildfly:
   ## @param wildflyPassword WildFly user password
   ## Defaults to a random 10-character alphanumeric string if not set
   ##
-  wildflyPassword: "test"
+  wildflyPassword: "mestest1234"
```

Comment:
you'll have to rely on `extraEnvVarsSecret` and `WILDFLY_PASSWORD` to avoid harding it in a public repo :wink:

see: https://github.com/bitnami/containers/tree/main/bitnami/wildfly#configuration

Reply:
The issue is that the WILDFLY_PASSWORD is hardcoded in the chart and takes precedence over https://github.com/bitnami/charts/blob/ad308c2fcc162182933038eeff9a8880c0923991/bitnami/wildfly/templates/deployment.yaml#L153-L156.

Reply:
Could we do this with a kustomization patch?

Reply:
there. I see. https://github.com/bitnami/charts/blob/ad308c2fcc162182933038eeff9a8880c0923991/bitnami/wildfly/templates/deployment.yaml#L129-L133

It sucks 🫏 🎱

Reply:
> Could we do this with a kustomization patch?

Allowing kustomize would be lovely but it requires quite a bunch of changes on the appset from what I've seen.

Reply:
or keep it empty and it'll be autogenerated by helm.